redash/cli/groups.py

```diff
@@ -119,4 +119,7 @@ def list_command(organization=None):
 
         members = models.Group.members(group.id)
         user_names = [m.name for m in members]
-        print("Users: {}".format(", ".join(user_names)))
+        if user_names:
+            print("Users: {}".format(", ".join(user_names)))
+        else:
+            print("Users:")
```

Comment:
Avoid output `Users: `(With a space suffix)

Reply:
In general best to try to keep unrelated refactoring out of version bump PRs (since they're already pretty large) but this isn't a huge deal.

Reply:
https://github.com/getredash/redash/pull/6138/files#diff-4e8715c7a425ee52e74b7df4d34efd32e8c92f3e60bd51bc2e1ad5943b82032eL316 

The trailing white space `W291` here will be removed by the formatting tool.